### PR TITLE
[Release 1.8] Use new quote endpoint for USD estimations

### PR DESCRIPTION
**Pull request by W3stside**
Uses new quote endpoint for getting USD price against USDC

## Testing
1. check console networks tab that quote is called if price strategy is cowswap
2. price strategy can be checked in logs under filter `[useGetGpPriceStrategy::GP Price Strategy]::`

### CODE REVIEW

**Comment by elena-zh** [comment]
I have got a hard error when selected a token pair in Rinkeby (trade 1 WETH per DAI)
image

File: src/custom/hooks/useUSDCPrice/index.ts [hunk 1]
```diff
@@ -76,38 +82,45 @@ export default function useUSDCPrice(currency?: Currency) {
 
   useEffect(() => {
     const isSupportedChain = supportedChainId(chainId)
-    if (!isSupportedChain || !currency || !amountOut || !stablecoin) return
+    if (!isQuoteLoading || !isSupportedChain || !sellTokenAddress || !sellTokenDecimals) return
 
-    const params = {
-      baseToken: stablecoin.address,
-      quoteToken: currency.wrapped.address,
+    const baseAmount = STABLECOIN_AMOUNT_OUT[isSupportedChain]
```
Comment (alfetopito) [inline src/custom/hooks/useUSDCPrice/index.ts:87]: Not from this PR but `isSupportedChain` is not a flag, is it?
Would you mind updating the name?

**Review by alfetopito — COMMENTED** [review]
Code looks ok, but while testing I got the same issue as Elena on Rinkeby

Screenshot from 2021-12-21 08-43-56
Screenshot from 2021-12-21 08-43-17

It's using COWSWAP strategy and I can see `quote` api calls.

On xDai for example did not get an error, but I did get the price stuck on "loading".
Screenshot from 2021-12-21 08-47-14
Screenshot from 2021-12-21 08-47-10

After I changed the input amount though the change was immediate and I did get a price

**Review by alfetopito — APPROVED** [review]
Working great now!

One small suggestion: log the failures as warnings instead of errors?
Screenshot from 2021-12-21 19-51-39

**Review by elena-zh — APPROVED** [review]
LGTM!

However, I agree with @alfetopito to make red logs as yellow warnings
image

**Pull request merged**